platforms/core-configuration/model-core/src/main/java/org/gradle/api/internal/tasks/NodeExecutionContext.java

```diff
@@ -25,4 +25,8 @@ public interface NodeExecutionContext {
      * Locates the given execution service.
      */
     <T> T getService(Class<T> type) throws ServiceLookupException;
+
+    default boolean isFallbackContext() {
```

Comment:
💭 What is a fallback context?  Can you add some javadoc to explain this concept?

Reply:
Yeah, I am not super happy with this name, either. Given that this is about node execution, maybe using something like `isPartOfExecutionGraph()` would be better.
I called it `FallbackContext`, since when the node is not executed as part of an execution graph, is falls back to using a "global" context as fallback for the node context.